src/utils/is-enum-value.ts

```diff
@@ -2,7 +2,7 @@
  * Type guard to check to see if the given value is a valid value for the enum.
  */
 export function isEnumValue<T>(enumType: T, value: unknown): value is T[keyof T] {
-   return (Object.keys(enumType) as Array<keyof T>)
+   return (Object.keys(enumType as {}) as Array<keyof T>)
```

Comment:
`as` casts can be dangerous. As such, we avoid them if at all possible. Would adjusting the function signature to the following example remove the need for the cast?

```ts
export function isEnumValue<T extends object>(enumType: T, value: unknown): value is T[keyof T] {
```